src/Uno.UI/UI/Xaml/DependencyPropertyDetailsCollection.Bindings.cs

```diff
@@ -14,8 +14,8 @@ namespace Windows.UI.Xaml
 	/// </summary>
 	partial class DependencyPropertyDetailsCollection
 	{
-		private ImmutableList<BindingExpression> _bindings = ImmutableList<BindingExpression>.Empty;
-		private ImmutableList<BindingExpression> _templateBindings = ImmutableList<BindingExpression>.Empty;
+		private List<BindingExpression> _bindings;
+		private List<BindingExpression> _templateBindings;
```

Comment:
ImmutableList would allocate a new array, copy over, from the old to the new, and wrap the newly allocated array in a newly allocated ImmutableList. That looks very expensive. I don't see these to be exposed outside of this class, so it should be good as a regular List, and more performant.